android/services/mongodb-remote-perftests/src/androidTest/java/com/mongodb/stitch/android/services/mongodb/performance/SyncPerformanceIntTestsHarness.kt

```diff
@@ -189,8 +190,13 @@ class SyncPerformanceIntTestsHarness : BaseStitchAndroidIntTest() {
 
                         afterEach(ctx, numDoc, docSize)
                     } catch (e: Exception) {
-                        runResult.failures.add(FailureResult(iter, e.localizedMessage,
+                        var failureMessage = e.toString()
+                        if (e.localizedMessage != null) {
```

Comment:
[style nit] can convert this to a ternary or elvis operator

Reply:
Oh cool. I did not know about elvis operators. I know kotlin doesn't have a ternary operator